docs/docs-source/docs/modules/run-cloudflow-gcp-marketplace/pages/deploy-cloudflow.adoc

```diff
@@ -0,0 +1,45 @@
+= Deploy Cloudflow on GCP Marketplace
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+== Deploying Cloudflow
+
+You have followed the xref:find-cloudflow-on-gcp-marketplace.adoc[previous step] of locating Cloudflow in the Marketplace catalog and now see the following screen:
+
+.The Cloudflow page on Marketplace
+image::cloudflow-main-gcp.png[Cloudflow page on Marketplace]
+
+Spend some time reading through the information on the page. When you are ready, click "Configure" to proceed to the deployment screen:
+
+.The Cloudflow Configuration and Deployment Page
+image::deploy.png[Cloudflow deployment page]
+
+On this page, you have the ability to configure a few parameters before deploying Cloudflow. First, you'll need a Google Kubernetes Engine (GKE) cluster.
+If you have an existing cluster, you may select it from the dropdown menu.
+Or you can quickly spin up a new cluster in your preferred cluster zone by clicking the "Create cluster" button.
+Note that your existing cluster may not be available for selection if it does not meet the resource requirements, in which case you'll also need to create a new cluster.
```

Comment:
Once we know the minimal requirements, please add it here. (in separate PR is fine)